app/src/androidTest/java/com/github/se/eventradar/messages/MessagesTest.kt

```diff
@@ -46,39 +39,17 @@ class MessagesTest : TestCase(kaspressoBuilder = Kaspresso.Builder.withComposeSu
   @RelaxedMockK lateinit var mockNavActions: NavigationActions
 
   private lateinit var mockMessageRepository: IMessageRepository
-  private lateinit var mockUserRepository: IUserRepository
+  private lateinit var mockUserRepository: MockUserRepository
   private lateinit var mockMessagesViewModel: MessagesViewModel
 
-  private val sampleMessagesList =
-      MutableStateFlow(
-          MessagesUiState(
-              userId = "1",
-              messageList =
-                  List(10) {
-                    MessageHistory(
-                        user1 = "1",
-                        user2 = "$it",
-                        latestMessageId = "1",
-                        user1ReadMostRecentMessage = false,
-                        user2ReadMostRecentMessage = true,
-                        messages =
-                            mutableListOf(
-                                Message(
-                                    sender = "$it",
-                                    content = "Test Message",
-                                    dateTimeSent = LocalDateTime.now(),
-                                    id = "1")))
-                  }))
-
   @Before
   fun testSetup() = runTest {
     mockMessageRepository = MockMessageRepository()
     mockUserRepository = MockUserRepository()
 
-    mockkStatic(FirebaseAuth::class)
-    every { FirebaseAuth.getInstance().currentUser!!.uid } returns "1"
+    mockUserRepository.updateCurrentUserId("1")
 
-    for (i in 0..9) {
+    for (i in 2..9) {
```

Comment:
The userId (of the current user in the test) is "1", so if I included "1", then it would be a chat with itself. So the easiest thing to do was to start at "2".

Reply:
Makes sense, maybe just add a comment. Then LGTM.